schema/configuration-structure.asn

```diff
@@ -34,25 +34,161 @@ BEGIN
+      stdPrivExtensions SEQUENCE OF CHOICE {
+         hypervisorDependent SEQUENCE OF CHOICE {
+            h H, -- H: Hypervisor
+            hypervisorDependent SEQUENCE OF CHOICE {
+               shcounterenw Shcounterenw OPTIONAL, -- Shcounterenw
+               shgatpa      Shgatpa      OPTIONAL, -- Shgatpa
+               shtvala      Shtvala      OPTIONAL, -- Shtvala
+               shvsatpa     Shvsatpa     OPTIONAL, -- Shvsatpa
+               shvstvala    Shvstvala    OPTIONAL, -- Shvstvala
+               shvstvecd    Shvstvecd    OPTIONAL, -- Shvstvecd
+               ssstateen    Ssstateen    OPTIONAL, -- Ssstateen: Supervisor-mode view of Smstateen
+               ...
+            } OPTIONAL, -- hypervisorDependent
+            ...
+         } OPTIONAL, -- hypervisorDependent
+
+         sm1p11    Sm1p11    OPTIONAL, -- Sm1p11: Machine Architecture v1.11
+         sm1p12    Sm1p12    OPTIONAL, -- Sm1p12: Machine Architecture v1.12
+         smepmp    Smepmp    OPTIONAL, -- Smepmp: PMP Enhancements for memory access and execution prevention on Machine mode
+         smstateen Smstateen OPTIONAL, -- Smstateen: State-Enable
+         ss1p11    Ss1p11    OPTIONAL, -- Ss1p11: Supervisor Architecture v1.11
+         ss1p12    Ss1p12    OPTIONAL, -- Ss1p12: Supervisor Architecture v1.12  ????Was it ratified in 2021???
+         sscofpmf  Sscofpmf  OPTIONAL, -- Sscofpmf: Count Overflow and Mode-Based Filtering
+         ssptead   Ssptead   OPTIONAL, -- Ssptead
+         sstc      Sstc      OPTIONAL, -- Sstc: Supervisor-mode Timer Interrupts
+         ssube     Ssube     OPTIONAL, -- Ssube: sstatus.ube=1 Big-endian user-space
+         sv32      Sv32      OPTIONAL, -- Sv32: Page-based Virtual Memory, 32-bit
+         sv39      Sv39      OPTIONAL, -- Sv39: Page-based Virtual Memory, 39-bit
+         sv48      Sv48      OPTIONAL, -- Sv48: Page-based Virtual Memory, 48-bit
+         sv57      Sv57      OPTIONAL, -- Sv57: Page-based Virtual Memory, 57-bit
+         svinval   Svinval   OPTIONAL, -- Svinval: Fine-Grained Address-Translation Cache Invalidation
+         svnapot   Svnapot   OPTIONAL, -- Svnapot: NAPOT Translation Contiguity
+         svpbmt    Svpbmt    OPTIONAL, -- Svpbmt: Page-Based Memory Types
+
+         ss1p12Dependent SEQUENCE OF CHOICE {
+            ss1p12    Ss1p12, -- Ss1p12: Supervisor Architecture v1.12  ????Was it ratified in 2021???
+            ss1p12Dependent SEQUENCE OF CHOICE {
+               svinval   Svinval   OPTIONAL, -- Svinval: Fine-Grained Address-Translation Cache Invalidation
+               ...
+            } OPTIONAL, -- ss1p12Dependent
+            ...
+         } OPTIONAL, -- ss1p12Dependent
+
+         ...
+      } OPTIONAL, -- stdPrivExtensions
+
+      nonStdExtensions SEQUENCE OF CHOICE {
+         nonStdExtension PrintableString, -- Prefixed with a single “X”, ordered alphabetically and separated from other multi-letter extension names by an underscore on the RISC-V ISA name strings 
```

Comment:
I thought this field is for a single extension. But the comment talks about separating extensions with an underscore.
Which is it?

Reply:
Yes, thank you for the Review Comment @timsifive .  nonStdExtension is for a single extension, yet the comment is talking about more than one extension.  I have modified the comment to not talk about multiple extensions.  Good catch